### DISCUSSION TITLE
How to disable new inlay_hints

### DISCUSSION THREAD

**Discussion by twschum** [VonHeikemen/lsp-zero.nvim#413] [General]
I want to disable the inlay hints (inline type information text) which should be possible via lspconfig option as follows, but this doesn't work. I can't figure out how to properly pass thru opts via `extend_lspconfig()` either. Thanks!
```
  opts = {
    inlay_hints = { enabled = true },
  },
```

**Comment by VonHeikemen** [comment]
Inlay hints should be disabled by default. lsp-zero doesn't interact with inlay hints either, there is no option to enable/disable it using the plugin.

In Neovim v0.10 or greater you can disable it like this:

```lua
vim.lsp.inlay_hint.enable(false)
```
